app/components/aktivitet-oppsummering/AktivitetOppsummering.tsx

```diff
@@ -1,7 +1,7 @@
 import { Alert } from "@navikt/ds-react";
 import classNames from "classnames";
-import type { TAktivitetType } from "~/models/aktivitet.server";
-import type { IRapporteringsperiode } from "~/models/rapporteringsperiode.server";
+import { type TAktivitetType } from "~/models/aktivitet.server";
```

Comment:
Her foretrekker jeg `{ type TAktivitetType }` isteden for  `type {TAktivitetType }`.  Synes at det er mer ryddig og lettere å skanne gjennom alle imports. 

😍
```
import { type TAktivitetType }
import { useField } from
import { Radio, RadioGroup }
```

vs

🫤
```
import type { TAktivitetType }
import { useField } from
import { Radio, RadioGroup }
```

Enda bedre eksempel er når vi importere funksjon eller konstant og types fra en fil, det er enklere å se hva som er  ts-types eller noe annen.

`import { SerializeFrom, type ActionFunctionArgs } from "@remix-run/node";`